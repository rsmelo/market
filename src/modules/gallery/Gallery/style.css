.gallery {
  display: flex;
  flex-direction: column;
  height: 100vh;
  width: 100%;
}

.content {
  display: flex;
  flex-flow: row wrap;
  flex-grow: 1;
  justify-content: center;
}
