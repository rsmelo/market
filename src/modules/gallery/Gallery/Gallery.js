import React from 'react'

import ProductList from '../ProductList'
import style from './style.css'

const Gallery = () => (
  <div className={style.gallery}>
    <div className={style.content}>
      <ProductList />
    </div>
  </div>
)

export default Gallery
