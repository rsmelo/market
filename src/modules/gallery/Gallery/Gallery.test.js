import React from 'react'
import renderer from 'react-test-renderer'

import Gallery from './Gallery'

jest.mock('../ProductList', () => 'ProductList')

describe('Gallery', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Gallery />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
