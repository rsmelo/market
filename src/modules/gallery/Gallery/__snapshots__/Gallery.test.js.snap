// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gallery renders correctly 1`] = `
<div
  className={undefined}
>
  <div
    className={undefined}
  >
    <ProductList />
  </div>
</div>
`;
